dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
I don't think it's git forcing the line ending changes, it's whatever package creates the dist files.  If it was git, it would be changing pretty much everything, not just the generated stuff.